src/tractusx_sdk/dataspace/controllers/connector/v0_9_0/edr_controller.py

```diff
@@ -0,0 +1,44 @@
+#################################################################################
+# Eclipse Tractus-X - Software Development KIT
+#
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the
+# License for the specific language govern in permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+from .dma_controller import DmaController
+from ..mixins import CreateControllerMixin, GetAllControllerMixin, DeleteControllerMixin
+from ....models.connector.v0_9_0 import ContractNegotiationModel
+
+
+class EdrController(CreateControllerMixin, GetAllControllerMixin, DeleteControllerMixin, DmaController):
+    """
+    Concrete implementation of the EdrController for the Connector v0.9.0 Data Management API.
+
+    This class overrides the create method in order to ensure the correct class types are used, instead of the generic ones.
+    """
+
+    endpoint_url = "/v3/edrs"
+
```

Comment:
```suggestion
    def post_request(self, queryspec: QuerySpecModel, **kwargs):
        return self.adapter.post(url=f"{self.endpoint_url}/request", **kwargs)

```
Could you add the `/v3/edrs/request` endpoint in the EDR controller, using the QuerySpec model as input? It's documented here in the API: https://eclipse-tractusx.github.io/api-hub/tractusx-edc/0.9.0/control-plane/swagger-ui/#/Control%20Plane%20EDR%20Api/requestEdrEntriesV3

This is related to the steps described here: https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/feat/update-edc-to-release-25-03/docs/user/guides/data-exchange/consume-data.md#query-edrs-request

Reply:
That request is already implemented by the `GetAllControllerMixin`, which is inherited by the `EdrController` class

Reply:
My bad, I missed those methods.